views/js/qtiCreator/widgets/interactions/matchInteraction/states/Map.js

```diff
@@ -18,11 +18,12 @@ define([
 
         //finally, apply defined correct response and response mapping:
         responseWidget.setResponse(interaction, _.values(response.getCorrect()));
-
-        matchInteractionArea
-            .prop('disabled', true)
-            .prop('checked', false)
-            .addClass('disabled');
+            if(!interaction.metaData.responseMappingMode) {
```

Comment:
Do we have case when `interaction.metaData.responseMappingMode === false`?
if MAP_RESPONSE is used, then this Map widget will be activated
I think we can just remove this code

Reply:
I thought the same and coudn't find a case where it was used, but I thought that I might be missing something... 
If you can't find any use for it either I'll remove it.